src/internals.ts

```diff
@@ -31,11 +31,11 @@ function pascalCaseAll<T extends string[]>(words: T) {
 /**
  * Removes all the elements matching the given condition from a tuple.
  */
-type Filter<tuple, cond, output extends any[] = []> = tuple extends [
+type Reject<tuple, cond, output extends any[] = []> = tuple extends [
```

Comment:
`Filter` would keep elements where predicate is true, `Reject` will drop elements where predicate is true.